base/src/moditect/module-info.java

```diff
@@ -4,7 +4,10 @@
 
     requires com.fasterxml.jackson.core;
     requires com.fasterxml.jackson.databind;
-    requires javax.ws.rs.api;
+    //Allow multiple implementations of ws.rs
+	requires static javax.ws.rs.api;
+	requires static java.ws.rs;
+	requires static jakarta.ws.rs.api;
```

Comment:
Are all 3 of these required? It would seem first and third were synonyms or... ?

Reply:
@GedMarc was hoping to clear this up, before releasing 2.10.2. So... are those all really needed? (wrt 1st and 3rd entries)